packages/progress-ring/src/__stories__/ProgressRing.stories.js

```diff
@@ -13,15 +14,19 @@ const storybook = storiesOf("ProgressRing", module);
 storybook.add(
   "determinate",
   withInfo(infoOptions)(() => (
```

Comment:
bit of a nitpick, but you can exclude `KnobbedThemeProvider` from the `storybooks` props info at the bottoms with: `withInfo({...infoOptions, propTablesExclude: [KnobbedThemeProvider]})`